projects/find/find.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
I don't understand what you mean. Os library is useful to print a path or a link.